From ce32ca99a5eb0319bc4b4c4dc3deefdafcfd67a6 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Mon, 18 Oct 2021 17:42:00 +0200
Subject: [PATCH 190/446] usb: typec: anx7688: Add support for i2c bus supply

Pinephone has separate supply for the i2c bus of the anx7688.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 drivers/usb/typec/anx7688.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/typec/anx7688.c b/drivers/usb/typec/anx7688.c
index a4549dc68..8786b5ea4 100644
--- a/drivers/usb/typec/anx7688.c
+++ b/drivers/usb/typec/anx7688.c
@@ -159,6 +159,7 @@ static const char * const anx7688_supply_names[] = {
         "dvdd18",
         "avdd10",
         "dvdd10",
+        "i2c",
         "hdmi_vt",
 
         "vconn", // power for VCONN1/VCONN2 switches
-- 
2.31.1

